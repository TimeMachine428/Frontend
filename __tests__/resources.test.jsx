import React from 'react';
import Resources from '../src/js/pages/Resources.jsx';
import renderer from 'react-test-renderer';

test('Resources page content',() =>{
	const component = renderer.create(
		<Resources />,
	);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
})

