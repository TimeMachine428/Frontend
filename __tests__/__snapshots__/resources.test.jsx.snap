// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resources page content 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <h1>
      Resources
    </h1>
  </div>
  <div
    className="row"
  >
    <div
      className="col-sm-4"
    >
      <h3>
        Programming Languages
      </h3>
      <a
        className="badge badge-light"
        href="https://www.learnpython.org/"
        target="_blank"
      >
         Python 
      </a>
       
      <br />
      <a
        className="badge badge-light"
        href="http://www.learncpp.com/"
        target="_blank"
      >
         C++ 
      </a>
       
      <br />
      <a
        className="badge badge-light"
        href="http://www.learncs.org/"
        target="_blank"
      >
         C# 
      </a>
       
      <br />
      <a
        className="badge badge-light"
        href="http://www.learn-js.org/"
        target="_blank"
      >
         JavaScript 
      </a>
       
      <br />
      <a
        className="badge badge-light"
        href="http://www.learnrubyonline.org/"
        target="_blank"
      >
         Ruby 
      </a>
       
      <br />
    </div>
    <div
      className="col-sm-4"
    >
      <h3>
        Data Structures
      </h3>
      <a
        className="badge badge-light"
        href="https://www.cs.cmu.edu/~adamchik/15-121/lectures/Linked%20Lists/linked%20lists.html"
        target="_blank"
      >
         Linked List 
      </a>
       
      <br />
      <a
        className="badge badge-light"
        href="https://www.hackerearth.com/practice/data-structures/stacks/basics-of-stacks/tutorial/"
        target="_blank"
      >
         Stack 
      </a>
       
      <br />
      <a
        className="badge badge-light"
        href="https://www.hackerearth.com/practice/data-structures/queues/basics-of-queues/tutorial/"
        target="_blank"
      >
         Queue 
      </a>
       
      <br />
      <a
        className="badge badge-light"
        href="https://www.topcoder.com/community/data-science/data-science-tutorials/introduction-to-graphs-and-their-data-structures-section-1/"
        target="_blank"
      >
         Graph 
      </a>
       
      <br />
      <a
        className="badge badge-light"
        href="http://www.cs.cmu.edu/~clo/www/CMU/DataStructures/Lessons/lesson4_1.htm"
        target="_blank"
      >
         Trees 
      </a>
       
      <br />
      <a
        className="badge badge-light"
        href="https://www.hackerearth.com/practice/data-structures/hash-tables/basics-of-hash-tables/tutorial/"
        target="_blank"
      >
         Hash Table 
      </a>
       
      <br />
    </div>
    <div
      className="col-sm-4"
    >
      <h3>
        Algorithms
      </h3>
      <a
        className="badge badge-light"
        href="https://www.hackerearth.com/practice/algorithms/graphs/breadth-first-search/tutorial/"
        target="_blank"
      >
         Breadth First Search 
      </a>
       
      <br />
      <a
        className="badge badge-light"
        href="https://www.hackerearth.com/practice/algorithms/graphs/depth-first-search/tutorial/"
        target="_blank"
      >
         Depth First Search 
      </a>
       
      <br />
      <a
        className="badge badge-light"
        href="https://www.geeksforgeeks.org/greedy-algorithms-set-2-kruskals-minimum-spanning-tree-mst/"
        target="_blank"
      >
         Kruskal's Algorithm 
      </a>
       
      <br />
      <a
        className="badge badge-light"
        href="https://www.geeksforgeeks.org/greedy-algorithms-set-9-boruvkas-algorithm/"
        target="_blank"
      >
         Boruvka's Algorithm 
      </a>
       
      <br />
      <a
        className="badge badge-light"
        href="http://lcm.csa.iisc.ernet.in/dsa/node183.html"
        target="_blank"
      >
         Prim's Algorithm 
      </a>
       
      <br />
      <a
        className="badge badge-light"
        href="http://math.mit.edu/~rothvoss/18.304.3PM/Presentations/1-Melissa.pdf"
        target="_blank"
      >
         Dijkstra's Algorithm 
      </a>
       
      <br />
      <a
        className="badge badge-light"
        href="https://www.hackerearth.com/practice/algorithms/greedy/basics-of-greedy-algorithms/tutorial/"
        target="_blank"
      >
         Greedy Algorithms 
      </a>
       
      <br />
      <a
        className="badge badge-light"
        href="https://www.topcoder.com/community/data-science/data-science-tutorials/dynamic-programming-from-novice-to-advanced/"
        target="_blank"
      >
         Dynamic Programming 
      </a>
       
      <br />
    </div>
  </div>
</div>
`;
